components/create_post/create_post.jsx

```diff
@@ -327,6 +327,7 @@ class CreatePost extends React.PureComponent {
         this.lastChannelSwitchAt = 0;
         this.draftsForChannel = {};
         this.lastOrientation = null;
+        this.textbox = React.createRef();
```

Comment:
can you rename it for consistency with other PRs
```suggestion
        this.textboxRef = React.createRef();
```